src/udev/udevadm-settle.c

```diff
@@ -167,11 +167,6 @@ int settle_main(int argc, char *argv[], void *userdata) {
         if (r <= 0)
                 return r;
 
-        if (running_in_chroot() > 0) {
```

Comment:
why is this no longer necessary? please explaini in the commit message

Reply:
Nah, I thought it is a uid check, but checking chroot. I overlooked 'ch'...